Source/Classes/OverlayContainerViewController.swift

```diff
@@ -298,23 +269,21 @@ open class OverlayContainerViewController: UIViewController {
             overlayContainerViewStyleConstraint = overlayContainerView.heightAnchor.constraint(
                 equalToConstant: 0
             )
-            overlayContainerViewStyleConstraint?.priority = .defaultHigh
-            if !landscapeLayout {
-                overlayContainerViewBottomConstraint = overlayContainerView.bottomAnchor.constraint(
-                    greaterThanOrEqualTo: overlayTranslationView.bottomAnchor
-                )
-
-                overlayContainerViewBottomConstraint?.isActive = true
-            }
+            let bottomConstraint = overlayContainerView.bottomAnchor.constraint(
+                greaterThanOrEqualTo: overlayTranslationView.bottomAnchor
+            )
+            bottomConstraint.isActive = true
         }
+
+        translationHeightConstraint?.priority = .defaultHigh
+        overlayContainerViewStyleConstraint?.priority = .defaultHigh
```

Comment:
These constraints usually define the height of the overlay, I've reduced their priority so that they will be overridden by the `landscapeLayoutConstraints` when they are enabled.